pytest_httpx/_httpx_mock.py

```diff
@@ -86,8 +99,12 @@ def __str__(self) -> str:
             matcher_description += f" with {self.headers} headers"
             if self.content is not None:
                 matcher_description += f" and {self.content} body"
+            if self.json_content is not None:
```

Comment:
should be a elif if we go for only one body can be matched

Reply:
Good point